README.md

```diff
@@ -1,4 +1,18 @@
 # coppeliasim_run
+This package runs coppeliasim as a ROS node using a system() call.
 
-## Usage
-This package runs coppeliasim _within_ a ROS node, so you can run it from a launch file.
+### Usage
+
+```
+# to launch coppeliasim as a ros-node from a terminal 
+rosrun coppeliasim_run start_coppeliasim [arguments]
+```
+
+#### Node Arguments: 
+- -h: runs CoppeliaSim in headless mode (i.e. without any GUI)
+- -sXXX: automatically start the simulation. XXX is simulation time in msec [optional]
+- -q: automatically quits CoppeliaSim after the first simulation run ended
+- -XXX.ttt: loads a CoppeliaSim scene
+- -XXX.ttm: loads a CoppeliaSim model
+- -XXX.brs: loads an XReality scene
+- -XXX.brm: loads an XReality model
```

Comment:
Suggestion: use a [code block](https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/creating-and-highlighting-code-blocks) to list these instead of an unordered list.

Github doesn't allow me to suggest a code-block (seems code-block in code-block isn't supported).